React State 2: Triple Count Challenge (#165)
neuefische/web-exercises: sessions/react-state-2/triple-count/README.md

# React State 2: Triple Count

The [`src/components/Counter.js`](./src/components/Counter.js) file offers a simple `count` state, a button to increment this state and a `handleIncrement` function.

> Clicking the button should increment the `count` by 3. Currently, however, it's incremented by 1, although the `handleIncrement` function calls `setCount(count + 1)` three times.
> 
> Of course, you could replace the two `setCount` calls by one `setCount(count + 3)`, but there are situations where this is not useful or even possible.
> 
> Fortunately, there is another solution!

Reviewer: ```suggestion
Clicking the button should increment the `count` by 3. Currently, however, it's incremented by 1, although the `handleIncrement` function calls `setCount(count + 1)` three times.

Of course, you could replace the two `setCount` calls by one `setCount(count + 3)`, but let's imagine that is not possible.

Fortunately, there is another solution!
```

Author: ```suggestion
Clicking the button should increment the `count` by 3. Currently, however, it's incremented by 1, although the `handleIncrement` function calls `setCount(count + 1)` three times.

Of course, you could replace the two `setCount` calls by one `setCount(count + 3)`, but let's imagine that this is not possible.

Fortunately, there is another solution!
```

## Task

> Fix the `handleIncrement` function so that it increments the `count` state by 3. Make sure to use `setCount` three times.
> 
> You can use the following hint as guideline:
> 
> - How can you update state based on the previous state?

Reviewer: ```suggestion
Fix the `handleIncrement` function so that it increments the `count` state by 3. For the sake of this challenge, make sure to still call `setCount` three times, incrementing by 1 each time.

You can use the following hint as guideline:

- How can you update state based on the previous state?
```
